### PR TITLE
[backend] add finalRating to factchain NFT attributes

### PR DESCRIPTION
**Pull request by Haypierre**

### CODE REVIEW

File: fc-community-backend/apps/cli/src/commands.ts [hunk 1]
```diff
@@ -133,29 +140,34 @@ export const finaliseNotes = async (
...
   postUrl: string,
   creator: string,
+  finalRating: number,
 ): Promise<ContractTransactionResponse> => {
   const fc = new FactChainBackend(config);
   return await fc.mintNote721({
     postUrl: postUrl,
     creatorAddress: creator,
     content: text,
+    finalRating: finalRating,
```
Comment (flocks) [inline fc-community-backend/apps/cli/src/commands.ts:170]:
```suggestion
    finalRating,
```

File: fc-community-backend/apps/cli/src/commands.ts [hunk 2]
```diff
@@ -133,29 +140,34 @@ export const finaliseNotes = async (
...
   console.log(responses);
+  console.log(`minted ${notesToFinalise.length} NFTs`);
   return responses;
 };
 
 export const mintNote = async (
   text: string,
   postUrl: string,
   creator: string,
+  finalRating: number,
```
Comment (flocks) [inline fc-community-backend/apps/cli/src/commands.ts:163]: It feels like you could also pass a single object with all the fieds:
```js
{ test: string; postUrl: string; creator: string; finalRating: number }
```
This way you can just 
```
 mintNote(options)
 ``` 
 in your [CLI command parser ](https://github.com/factchain/factchain-community/pull/53/files#diff-ee3f95f82ca25236f75b955ea5783e3ac50a529e063a29e9a103c4a9067327a5L108)

Comment (flocks) [reply fc-community-backend/apps/cli/src/commands.ts:163]: but that's a detail

File: fc-community-backend/apps/cli/src/commands.ts (hunk 1 above)
Comment (Haypierre) [reply fc-community-backend/apps/cli/src/commands.ts]: Thanks, I didn't know that :))

**Pull request merged**